1. table.sql

```diff
@@ -0,0 +1,23 @@
+use ArmyDB 
+go 
+drop table if exists Soldiers 
+go 
+
+create table dbo.Soldiers (
+    SoldierID int not null identity primary key,
+    FirstName varchar (30) not null constraint first_name_cannot_be_blank check (FirstName <> ''),
+    LastName  varchar (30) not null constraint last_name_cannot_be_blank check (LastName <> ''),
+    SSN char (10) not null constraint ck_ssn_10_characters check (len (SSN) = 10)
```

Comment:
`len(SSN)` is not enough when using `char(10)`, as this adds extra spaces and it will always be 10 characters. Remove trailing/leading spaces and then check the `len()`

Reply:
I'm not quite sure what you mean. Can you explain?

Reply:
Can you tell me how to do it?

Reply:
Sorry, I thought it was necessary, but I tried it and saw that SQL is restricting to insert even when inserting spaces.